wai-aria/role/roles.html

```diff
@@ -103,6 +103,8 @@
   // "rowgroup" -> ./grid-roles.html
   // "rowheader" -> ./grid-roles.html
   "scrollbar",
+  "sectionheader",
+  "sectionfooter",
```

Comment:
Since all implementations have a different mappings (to `generic`), should this be a separate PR from the ATTAcomm tests?

Reply:
Also, it be in a `.tentative` file until there is a spec change landed and two passing implementations.